NEWS.md

```diff
@@ -5,6 +5,12 @@
 * `seq_linter()` additionally lints on `1:n()` (from dplyr) 
   and `1:.N` (from data.table)  (#1396, @IndrajeetPatil).
 
+## Bug fixes
+
+* `get_source_expression()` no longer omits trailing non-code lines from knitr files (#1400, @AshesITR).  
```

Comment:
still hoping to clarify if it's a typo -- shouldn't this have a trailing `s`?

Reply:
Oh, good catch! Yes, it should.